.github/workflows/build.yml

```
@@ -67,7 +67,7 @@ jobs:
         brew install libuv swig pkgconfig jsoncpp
     - name: cmake configure
       working-directory: ${{env.BuildDir}}
-      run: cmake "${{github.workspace}}" "-DCMAKE_BUILD_TYPE=${BuildType}" "-DCMAKE_INSTALL_PREFIX=${InstallPrefix}" -DRUNTIME_CHECK=${{matrix.runtimeCheck}} -DBUILD_BINDINGS=python3 ${{matrix.cmake_extra}}
+      run: cmake "${{github.workspace}}" "-DCMAKE_BUILD_TYPE=${BuildType}" "-DCMAKE_INSTALL_PREFIX=${InstallPrefix}" -DRUNTIME_CHECK=${{matrix.runtimeCheck}} -DBUILD_BINDINGS=python,python3 ${{matrix.cmake_extra}}
```

```suggestion
      run: cmake "${{github.workspace}}" "-DCMAKE_BUILD_TYPE=${BuildType}" "-DCMAKE_INSTALL_PREFIX=${InstallPrefix}" -DRUNTIME_CHECK=${{matrix.runtimeCheck}} -DBUILD_BINDINGS="python;python3" ${{matrix.cmake_extra}}
```

and it has to be a semicolon, which is a bit tricky, because semicolon is a command separator in linux shell, therefore I also added quotes, to be sure I don't end up executing two shell commands separated by the semicolon...

this is documented in a comment in CMakeLists.txt, but nowhere else I could find; so I guess that's something to mention in the jira too... feature needs 1) clear error when you specify binding that does not exist, 2) documentation

---

@SudhanAruna this is the reasons no tests are running

---

> @SudhanAruna You still did not fix this. And please create a jira for the problem, as I wrote before.

Sorry I missed this notification regarding this comment.  By "create a jira for the problem" you mean to create  issue for issue for this  problem?

---

Yes, exactly. So that we don't forget to fix it later.